src/App.js

```diff
@@ -0,0 +1,112 @@
+import {useState} from "react";
+import 'react-tabs/style/react-tabs.css';
+import './App.css';
+
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+
+import Header from "./Header.js";
+import Footer from "./Footer.js";
+import Alert from "./Alert.js";
+import Tasks from "./Tasks.js";
+
+const initialData = [
+    {
+        id: 517,
+        text: "Eat Lunch"
+    },
+    {
+        id: 787,
+        text: "Buy new John Grisham book"
+    },
+    {
+        id: 911,
+        text: "Buy new John Grisham book"
+    },
+    {
+        id: 231,
+        text: "Buy new John Grisham book"
+    },
+    {
+        id: 589,
+        text: "Buy new John Grisham book"
+    }
+];
+
+function App() {
+    const [taskList, setTaskList] = useState(initialData);
+    const [completedTaskList, setCompletedTaskList] = useState([517, 911, 231]);
```

Comment:
This will not work out well when you move to Firestore.  You'll want the completed state to be stored in each task, not in separate state information.